### ISSUE TITLE
Raty is not defined

### ISSUE THREAD

**Issue by ngohuytrieu** [wbotelhos/raty#251]
I am trying to integrate this into my website from jsdelivr
```html
<script src="https://cdn.jsdelivr.net/npm/raty-js@4.1.0/src/raty.min.js"></script>
<link href="https://cdn.jsdelivr.net/npm/raty-js@4.1.0/src/raty.min.css" rel="stylesheet">
```

But when I open chrome console and try to run `new Raty()` and get this error
```
Uncaught ReferenceError: Raty is not defined
```

This not only happens to Raty but many other libraries that I want to integrate into my website. Any ideal why? Thank you!

**Comment by ajiho** [comment]
The 4. x version is completely unusable, please revert back to the 3. xJquery version

**Comment by wbotelhos** [comment]
Hi @ngohuytrieu , it's an import issue on your side. Please, test it into your code, not in the console and tell me. If it persists, please, create an example and I'll be happy to review.

**Comment by wbotelhos** [comment]
@ajiho , I'm sorry that it's not work for you. I'm using it in productions and it's working well, and so far, no one else said it's completely unusable, so maybe it's something you're doing wrong on your site.

When you have a specific problem, please report here so I can help you.

**Comment by ajiho** [comment]
@wbotelhos My problem has been resolved ,Thank you again [#252](https://github.com/wbotelhos/raty/issues/252)
